.github/workflows/post-release.yml

```diff
@@ -0,0 +1,126 @@
+name: Post-Release
+
+permissions:
+  contents: read
+
+on:
+
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      alt_version:
+        required: false
+        description: Provide expected semver. If empty VERSION in repo will be used
+        default: ''
+
+env:
+  USE_VERSION: ${{ github.event.inputs.alt_version }}
+
+jobs:
+  installer_check:
+    name: Installer check
+    strategy:
+      matrix:
+        version: [none, stable, semver]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set the expected fluvio version
+        run: |
+          if [[ -z "${{ env.USE_VERSION }}" ]]; then
+            echo "EXPECTED_VERSION=$(cat VERSION)" | tee -a $GITHUB_ENV
+          else
+            echo "EXPECTED_VERSION=${{ github.event.inputs.alt_version }}" | tee -a $GITHUB_ENV
+          fi
+          echo "EXPECTED_VERSION: $EXPECTED_VERSION"
+
+      - name: Curl Install - no version
+        if: matrix.version == 'none'
+        run: curl -fsS https://packages.fluvio.io/v1/install.sh | bash | tee /tmp/installer.version
+      - name: Curl Install - stable 
+        if: matrix.version == 'stable'
+        run: curl -fsS https://packages.fluvio.io/v1/install.sh | VERSION=stable bash | tee /tmp/installer.version
+      - name: Curl Install - Version number
+        if: matrix.version == 'semver'
+        run: curl -fsS https://packages.fluvio.io/v1/install.sh | VERSION=$EXPECTED_VERSION bash | tee /tmp/installer.version
+
+      - name: Verify installer output
+        run: |
+            INSTALLER_VERSION=$(cat /tmp/installer.version | grep "Downloading Fluvio" | awk '{print $5}')
+            if [ "$INSTALLER_VERSION" = "$EXPECTED_VERSION" ]; then
+              echo "✅ Installer version check passed: $EXPECTED_VERSION";
+            else
+              echo "❌ Installer version check failed";
+              echo "Version reported by installer: $INSTALLER_VERSION";
+              echo "Expected version: $EXPECTED_VERSION";
+              exit 1;
+            fi
+
+  start_cluster:
+    name: Start cluster test
+    needs: installer_check
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        cluster_type: [local, k8s]
+    env:
+      FLV_SOCKET_WAIT: 600
+    steps:
+      - uses: nolar/setup-k3d-k3s@v1
+        if: matrix.cluster_type == 'k8s'
+      - name: Install Fluvio
+        run: curl -fsS https://packages.fluvio.io/v1/install.sh | bash
+      - name: Start local cluster
+        if: matrix.cluster_type == 'local'
+        run: ~/.fluvio/bin/fluvio cluster start --local
```

Comment:
similar to above just compute `--local` using cluster type

Reply:
I don't quite understand the feedback. What am I supposed to compute?

The purpose of this matrix test is to verify the default behavior for starting a cluster, and the two modes (local, or k8) have different CLI commands (the `--local` flag), so I put them in different arms.

This test is one of the main motivators for #1433